packages/office-ui-fabric-react/src/components/FocusZone/FocusZone.tsx

```diff
@@ -623,10 +633,17 @@ export class FocusZone extends BaseComponent<IFocusZoneProps, {}> implements IFo
       ) {
 
         distance = activeRect.right - targetRect.right;
+      } else {
+        if (!shouldWrap) {
+          distance = -2;
```

Comment:
I don't quite understand the -2

Reply:
It's just one less than -1 :) I needed to use a more negative number (for the distance from center) to keep the loop going in. I pulled out a const for both the upper value (that already existed) and the lower value that I added. The large value picks  the element as a potential element, while the negative number is for not picking that element